src/appendices/string-method-examples/trim-examples.rst

```diff
@@ -0,0 +1,48 @@
+.. _string-trim-examples:
+
+**trim** Examples
+=================
+
+The general syntax for this method is:
+
+.. sourcecode:: js
+
+   stringName.trim();
+
+.. index::
+   single: whitespace; leading
+   single: whitespace; trailing
+
+This method returns a copy of the string with any leading or trailing whitespace removed. **Leading whitespace** consists of all whitespace characters---such as spaces or tabs---appearing *before* the first non-whitespace character. **Trailing whitespace** consists of all whitespace characters appearing *after* the last non-whitespace character.
```

Comment:
The definitions of leading and trailing whitespace seem unnecessary.  Perhaps just define **whitespace**.  The students will understand what leading and trailing mean.